common/djangoapps/enrollment/views.py

```diff
@@ -730,20 +733,20 @@ def post(self, request):
                 enterprise_api_client = EnterpriseApiServiceClient()
                 consent_client = ConsentApiServiceClient()
                 try:
-                    enterprise_api_client.post_enterprise_course_enrollment(username, unicode(course_id), None)
+                    enterprise_api_client.post_enterprise_course_enrollment(username, six.text_type(course_id), None)
```

Comment:
`text_type` was already imported directly in the file, `python-modernize` just didn't notice it.  Maybe use that and drop `import six`?

Reply:
Good catch, fixed